kube-prereq.adoc

```diff
@@ -8,28 +8,26 @@
 ////
 == Prerequisites
 
-Before you begin, have the following tools installed:
+Before you begin, the following tools must be installed:
 
 
-First, you need a containerization software for building containers. {kube} supports various container runtimes. You will use `Docker` in this guide. For installation instructions, refer to the https://docs.docker.com/install/[official Docker documentation^].
+First, you need a containerization software for building containers. {kube} supports various container runtimes. You will use `Docker` in this guide. For Docker installation instructions, refer to the https://docs.docker.com/install/[official Docker documentation^].
```

Comment:
Change
"For Docker installation instructions, refer to the https://docs.docker.com/install/[official Docker documentation^]."
To
"For Docker installation instructions, refer to the official https://docs.docker.com/install/[Docker documentation^]."

Reply:
Changed.